### PR TITLE
Fixed all known instances of plone.com (to plone.org) in plone/CMFPlone (Issue #3568)

### PR DESCRIPTION
**Pull request by rohnsha0**
Changes asked in Issue #3568

### CODE REVIEW

**Comment by rohnsha0** [comment]
@jenkins-plone-org please run jobs

File: Products/CMFPlone/browser/templates/plone-frontpage.pt
Comment (stevepiercy) [inline Products/CMFPlone/browser/templates/plone-frontpage.pt:42]: Let's change this completely. That link is not useful because it is marketing to someone who has already installed Plone and does not need to be sold on it again, and the page name no longer exists.

```suggestion
    <li>Get the latest <a class="link-plain" href="https://plone.org/what-is-plone/plone" target="_blank">news</a> about Plone.</li>
```

File: Products/CMFPlone/browser/templates/plone-overview.pt
Comment (stevepiercy) [inline Products/CMFPlone/browser/templates/plone-overview.pt:34]: I think it is appropriate to omit "success stories" as the reader is already sold on installing Plone and trying it.

File: Products/CMFPlone/browser/templates/plone-overview.pt
Comment (stevepiercy) [inline Products/CMFPlone/browser/templates/plone-overview.pt:36]: I pulled this from the `og:title` meta tag.

```suggestion
              title="Plone CMS: Open Source Content Management"
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
I've made a couple more suggestions. I think that someone from the Communications and Marketing Team should review as well.

File: Products/CMFPlone/browser/templates/plone-frontpage.pt
Comment (rohnsha0) [reply Products/CMFPlone/browser/templates/plone-frontpage.pt:42]: I'll do this one... This was my initial thought too but later decided not to change!

File: Products/CMFPlone/browser/templates/plone-overview.pt
Comment (rohnsha0) [reply Products/CMFPlone/browser/templates/plone-overview.pt:34]: But `success stories` option is already omitted! Sir...

**Comment by rohnsha0** [comment]
> I've made a couple more suggestions. I think that someone from the Communications and Marketing Team should review as well.

@plone/marketing-communications might be interested in reviewing!

File: Products/CMFPlone/browser/templates/plone-overview.pt
Comment (stevepiercy) [reply Products/CMFPlone/browser/templates/plone-overview.pt:34]: Yes, I was merely stating why I think it should be omitted, in case someone else notices it and wants it reinstated just for the sake of reinstatement. It should only be put back if it is reasonable to do so.

**Review by stevepiercy — APPROVED** [review]

File: Products/CMFPlone/browser/templates/plone-overview.pt
Comment (rohnsha0) [reply Products/CMFPlone/browser/templates/plone-overview.pt:34]: Okay! hahah😅
So shall I marks this as resolved? Or keep it like this only?

Comment (stevepiercy) [reply Products/CMFPlone/browser/templates/plone-overview.pt:34]: It is OK to resolve. It can be unresolved, if needed, until it is merged.

Comment (mtrebron) [reply Products/CMFPlone/browser/templates/plone-overview.pt:34]: Agree with replacing 'success stories' with 'documentation'

File: Products/CMFPlone/browser/templates/plone-overview.pt
Comment (mtrebron) [reply Products/CMFPlone/browser/templates/plone-overview.pt]: @skleinfeldt @rioksane do we want to emphasize the Plone _Community_ aspect here? Or like @stevepiercy suggests: synchronize the title tag for plone.org everywhere? Either works for me.

Comment (mtrebron) [reply Products/CMFPlone/browser/templates/plone-overview.pt]: Also see here: https://github.com/collective/plone.app.locales/blob/master/plone/app/locales/locales/plone.pot#L12297

Comment (stevepiercy) [reply Products/CMFPlone/browser/templates/plone-overview.pt]: Does this need to be removed?

https://github.com/collective/plone.app.locales/blob/master/plone/app/locales/locales/plone.pot#L12287-L12290

Comment (rohnsha0) [reply Products/CMFPlone/browser/templates/plone-overview.pt]: @stevepiercy I do think... As it shows the same `message-id` as changed `title` in (Products/CMFPlone/browser/templates/plone-overview.pt)

Comment (mtrebron) [reply Products/CMFPlone/browser/templates/plone-overview.pt]: Is this possible instead?

```
#. Default: "Plone.org"
#: CMFPlone/browser/templates/plone-overview.pt:35
#| msgid "label_plone_com_title"
msgid "label_plone_org_title"
msgstr ""
```

Comment (rohnsha0) [reply Products/CMFPlone/browser/templates/plone-overview.pt]: @mtrebron better!

Comment (mtrebron) [reply Products/CMFPlone/browser/templates/plone-overview.pt]: Sorry for the scattered thoughts: if it is only used in the plone-overview template, maybe better to remove altogether.

Comment (mtrebron) [reply Products/CMFPlone/browser/templates/plone-overview.pt]: We probably need some expert opinion from the translators team. @erral l ? I am not sure if plone.pot is generated in some way.

Comment (rohnsha0) [reply Products/CMFPlone/browser/templates/plone-overview.pt]: What to remove? Sir...

Comment (mtrebron) [reply Products/CMFPlone/browser/templates/plone-overview.pt]: I would suggest reverting this back to "Plone Community Home" and create a new separate issue in collective/plone.app.locales to deal with the .pot changes

Comment (rohnsha0) [reply Products/CMFPlone/browser/templates/plone-overview.pt]: Okay... I'll revert them soon!

File: Products/CMFPlone/browser/templates/plone-overview.pt
Comment (stevepiercy) [inline Products/CMFPlone/browser/templates/plone-overview.pt:36]: Please revert this change, per https://github.com/plone/Products.CMFPlone/pull/3607#discussion_r945289380

**Review by stevepiercy — CHANGES_REQUESTED** [review]

**Review by stevepiercy — APPROVED** [review]

File: Products/CMFPlone/browser/templates/plone-overview.pt
Comment (rohnsha0) [reply Products/CMFPlone/browser/templates/plone-overview.pt]: changed!

**Comment by rohnsha0** [comment]
@jenkins-plone-org please run jobs

File: Products/CMFPlone/browser/templates/plone-overview.pt
Comment (mauritsvanrees) [reply Products/CMFPlone/browser/templates/plone-overview.pt:34]: Seems good.

**Review by mauritsvanrees — APPROVED** [review]
Thanks. I will merge.

**Pull request merged**
